### PR TITLE
Display recent activity in homepage community cards

### LINKED ISSUE #731: Show recently active threads and community members on the homepage
We should create an endpoint that calculates and shows 1) recently active threads 2) recently active community members for each community, pass it through /api/status, and display the resulting info on the homepage:

image

This is similar to what many other forums do when they show e.g. `4 / wk` on a tag. We will probably want that within communities for each topic later, but it makes more sense to implement that when we start working on topics again in the future.

### PR DESCRIPTION
**Pull request by gdjohnson**
Closes #731.

## Description
image

This branch displays, on a rolling basis, the past month's new thread count, as well as recent active users.

The /status endpoint now counts the number of threads created in the past 30 days in a given community, as well as collecting the list of all users (in AddressInfo format) who have created or updated a thread, comment, or reaction in the past 30 days. This data is stored in `app.recentActivity`. Both pieces of data are displayed on the homepage's CommunityCards, as pictured above—as a count for threads, and a UserGallery for members.

Homepage cards need some CSS to stay in gridded alignment, but it might be best to leave that to @raykyri.

## How has this been tested?

I've manually inspected the data to ensure proper fetching/sorting is occurring for each chain/community. I've tested different time boundaries to see how different levels of activity (e.g. many recently active users) would affect card rendering. 

## Have proper tags been added (for bug, enhancement, breaking change)?
- [x] yes

## Does this PR affect any server routes?
Currently, we don't have tests set up for the `/status` endpoint.

### CODE REVIEW

**Review by raykyri — CHANGES_REQUESTED** [review]
Made a few copy changes, only one comment at this point:

- Could we make it so that communities on the homepage are sorted in descending order by number of recently active threads? This will require moving some the monthlyThreads calculation around but it'll make a really big difference in making the page feel more active.

File: client/scripts/views/pages/home/community_cards.ts [hunk 1]
```diff
@@ -5,8 +5,9 @@ import { Button, Icon, Icons, Card, Tag } from 'construct-ui';
 
 import app from 'state';
 import { link, pluralize } from 'helpers';
-import { NodeInfo, CommunityInfo } from 'models';
+import { NodeInfo, CommunityInfo, AddressInfo } from 'models';
 import { ChainIcon, CommunityIcon } from 'views/components/chain_icon';
+import UserGallery from '../../components/widgets/user_gallery';
```
[the remark this answers is not in the document]
Comment (gdjohnson) [reply client/scripts/views/pages/home/community_cards.ts:10]: yeah sorry that's my auto-import actin up, lemme see if I can tweak settings

**Comment by gdjohnson** [comment]
@raykyri Import cleaned up, and homepage sorting added

**Review by raykyri — APPROVED** [review]
LGTM

**Pull request merged**